### PR TITLE
[TF Hub] Set seed for input data generation and fix integer input data

### PR DESCRIPTION
**Pull request by rkazants**
**Details:** Set seed for input data generation and fix integer input data

**Ticket:** TBD

### CODE REVIEW

File: .github/workflows/linux.yml [hunk 1]
```diff
@@ -850,8 +850,6 @@ jobs:
 
       - name: TensorFlow Hub Tests - TF FE
         run: |
-          python3 -m pip install openvino --find-links=${{ env.INSTALL_DIR }}/tools
-
```
Comment (ilya-lavrenov) [inline .github/workflows/linux.yml:854]: why OpenVINO installation is removed?

Comment (rkazants) [reply .github/workflows/linux.yml:854]: this is redundant step. We have the same step above

File: tests/model_hub_tests/models_hub_common/test_convert_model.py [hunk 2]
```diff
@@ -7,6 +7,10 @@
 from openvino import convert_model
 from openvino.runtime import Core
 
+# set seed to have determenistic generated test data
+# to avoid sporadic issues due to input test data
+np.random.seed(56190)
```
Comment (mvafin) [inline tests/model_hub_tests/models_hub_common/test_convert_model.py:12]: This is a legacy function, please use generator
https://numpy.org/doc/stable/reference/random/generated/numpy.random.seed.html

Comment (rkazants) [reply tests/model_hub_tests/models_hub_common/test_convert_model.py]: Done

**Review by gkrivor — APPROVED** [review]

**Pull request merged**